model-server/requirements.txt

```diff
@@ -42,15 +42,15 @@ nbconvert==5.5.0
 nbformat==4.4.0
 notebook==5.7.8
 # Don't override system apt installed numpy and opencv-python on Raspbian.
-#numpy==1.17.3
-#opencv-python==4.1.2.30
-#opencv-python
+numpy==1.17.3
+# opencv-python==4.1.2.30
+opencv-python
```

Comment:
@bleggett - Comment out numpy and opencv-python due to Pi system apt installed working packages.

Reply:
@jbondpk My problem with this is that it works out of the box for the kit but doesn't work if you're using/testing locally - it looks like the clean way to solve this is with environment markers where we skip those deps unless we're on ARM.

Reply:
Done - seems to work.